sample/build.gradle

```diff
@@ -1,24 +1,19 @@
 apply plugin: 'com.android.application'
-apply plugin: 'me.tatarka.retrolambda'
 
 buildscript {
     repositories {
         mavenCentral()
     }
-
-    dependencies {
-        classpath 'me.tatarka:gradle-retrolambda:3.6.1'
-    }
 }
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion "25.0.3"
+    compileSdkVersion 28
+    buildToolsVersion "28.0.3"
 
     defaultConfig {
         applicationId "de.rheinfabrik.heimdall"
-        minSdkVersion 15
-        targetSdkVersion 25
+        minSdkVersion 21
```

Comment:
Do we need to raise the `minSdk` version?

Reply:
not really 😇